Sources/LiveKit/participant/RemoteParticipant.swift

```diff
@@ -4,8 +4,9 @@ import Promises
 
 public class RemoteParticipant: Participant {
 
-    init(sid: String, info: Livekit_ParticipantInfo?) {
+    init(sid: String, info: Livekit_ParticipantInfo?, room: Room) {
         super.init(sid: sid)
+        self.room = room
```

Comment:
Is this just for consistency in `init` signature with `LocalParticipant` as this new field does not seem to be used anywhere OR is it some interface/implementation making it necessary to change this signature?

Reply:
It's actually used in the parent Participant class, which I forgot to do before! So I fixed it now 😅

Reply:
Got it. Thank you!